### ISSUE TITLE
Create new Account: Root type missing

### ISSUE THREAD

**Issue by bosue** [frappe/erpnext#37508]
### Information about bug

When creating a new account from the “New“ button on the Charts of Accounts page, the “Root Type“ field is present only if “Is Group” is being checked.

For non-group accounts this however means saving the account is doomed to fail as “Root Type” turns out to be required:

https://github.com/frappe/erpnext/assets/46800703/ca785499-fdd5-4194-902b-e2046d1cf7c7

Now we could fix this, but then the next problem will be that the “Parent Account” field is missing, too.

As the Quick Entry Form is limited to 7 fields maximum, I think we should add “Parent Account”, skip the currency which may still be changed later, and fix the logic.

### Module

accounts

### Version

Frappe / ERPNext: dev

### Installation method

None

**Comment by GeorgeDeac** [comment]
Up, any updates on this?

**Comment by bosue** [comment]
Nope, but thanks for the reminder. I‘m quite busy this week, but will come back to it asap.

**Comment by swissbyte** [comment]
Im facing the same issue here... any updates on this?
I would like to be able to create root accounts using the WEB Frontend. It seems to be impossible

**Comment by L1lith04** [comment]
I’m working with ERPNext v15.x.x-develop and Frappe v15.x.x-develop and I’m still experiencing this issue as well. Are there any updates or potential fixes for this issue?

**Comment by diptanilsaha** [comment]
To add accounts using the `New` Button under a created Account, select the Account and click the `New` button. For child accounts, Root Type will be fetched from the Parent Account's Root Type.
